NeuroID/src/test/java/com/neuroid/tracker/NeuroIDClassUnitTests.kt

```diff
@@ -265,6 +288,16 @@ class NeuroIDClassUnitTests {
         assertErrorCount(1)
     }
 
+    @Test
+    fun testSetRegisteredUserID_failure() {
+        setNeuroIDMockedLogger(errorMessage = "Invalid UserID")
+
+        val value = NeuroID.getInstance()?.setRegisteredUserID("Bad User REGISTERED ID")
+
+        assertEquals(false, value)
```

Comment:
Could also add the same check as you did in the other test, just to ensure this event is not added to the queue on error `assertEquals(0, storedEvents.count())`, maybe also set the SDK to started, just to have a consistent test

Reply:
fixed